Lib/test/support/__init__.py

```diff
@@ -835,6 +835,9 @@ def dec(*args, **kwargs):
 # module name.
 TESTFN = "{}_{}_tmp".format(TESTFN, os.getpid())
 
+# Define the URL of a dedicated HTTP server for the network test.
```

Comment:
Please document that the URL must use clear-text HTTP: no redirection to encrypted HTTPS.

For example, http://www.python.org/ cannot be used: it uses a direction to HTTPS.